src/fsharp/fsi/fsiAnyCPU.exe.config

```diff
@@ -10,7 +10,7 @@
           culture="neutral"/>
         <bindingRedirect
           oldVersion="2.0.0.0-4.4.1.0"
-          newVersion="4.4.1.9055"/>
+          newVersion="4.4.1.0"/>
```

Comment:
Yes. The other side-effect is that now we will depend on VS loading all dependent VS binaries for language services/project system when in debugging mode.